napari/_qt/_qapp_model/_tests/utils.py

```diff
@@ -0,0 +1,25 @@
+import qtpy
+from qtpy.QtWidgets import QMenu
+
+
+def get_submenu_action(window_menu, submenu_text, action_text):
```

Comment:
Could we change `window_menu` because it could be confused for the menubar 'window' menu - though I know you are meaning `viewer.window.<menu>`.

`q_model_menu` is accurate but maybe a bit long?

Reply:
I think `qmodel_menu` could be fine :+1: